### ISSUE TITLE
wildfly component push fails with certain arguments

### ISSUE THREAD

**Issue by amitkrout** [openshift/odo#1695]
[kind/bug]

## What versions of software are you using?
- Operating System: Supported platform
- Output of `odo version`: master

## How did you run odo exactly?
```
$ oc new-projects test1
$ odo create wildfly wo-wait-javaee-git-test --project test1 --ref master --git https://github.com/lordofthejars/book-insultapp
$ odo url create gitrepo --port 8080
$ odo push -v 4
```
## Actual behavior
Stuck at 

<img width="1211" alt="Screen Shot 2019-05-06 at 2 22 14 PM" src="https://user-images.githubusercontent.com/19851115/57215410-69e35200-700a-11e9-921f-e4edc8b0dafa.png">

## Expected behavior
Should push successfully

**Comment by amitkrout** [comment]
@kadel ping

**Comment by amitkrout** [comment]
Pr #1634 should fix this

**Comment by amitkrout** [comment]
Unable to reproduce it on master.
Fixed via #1740
